devtools/cli/src/main/java/io/quarkus/cli/QuarkusCli.java

```diff
@@ -3,32 +3,26 @@
 import java.io.PrintWriter;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.LinkedList;
-import java.util.concurrent.Callable;
 
 import javax.inject.Inject;
 
-import io.quarkus.cli.core.BuildsystemCommand;
-import io.quarkus.cli.core.ExecuteUtil;
+import io.quarkus.cli.core.BuildSystemRunner;
 import io.quarkus.cli.core.QuarkusCliVersion;
 import io.quarkus.devtools.project.BuildTool;
 import io.quarkus.devtools.project.QuarkusProject;
 import io.quarkus.runtime.QuarkusApplication;
 import io.quarkus.runtime.annotations.QuarkusMain;
+import picocli.AutoComplete;
 import picocli.CommandLine;
 
 @QuarkusMain
-@CommandLine.Command(name = "qs", versionProvider = QuarkusCliVersion.class, subcommandsRepeatable = true, mixinStandardHelpOptions = true, subcommands = {
-        Build.class,
-        Clean.class, Create.class, CreateJBang.class, List.class, Platforms.class, Add.class, Remove.class, Dev.class,
-        CreateExtension.class })
+@CommandLine.Command(name = "quarkus", versionProvider = QuarkusCliVersion.class, subcommandsRepeatable = true, mixinStandardHelpOptions = true, subcommands = {
```

Comment:
So in the end, we decided against `qs`?

Reply:
I'm fine with it btw, just wanted to check.

Reply:
In the email thread (I think), it was stated we should just go with quarkus, as aliases are easy enough to add. 

Nothing has been "decided", this is literally an experiment so people can play with it (e.g. tab complete and help).